content/blobstore-ca-rotation.md

```diff
@@ -0,0 +1,239 @@
+!!! note
+    Applicable for director version 266.2.0+
+
+    Linux stemcells v3586.5+ (agent 2.83.0)
+
+    Windows 2016 stemcell v1709.9+, Windows 2012R2 v1200.20+ (agent 2.110.0)
+
+# Rotating the Blobstore CA
+
+Applies to a director with TLS enabled for the default DAV blobstore.
+
+Example director configuration enabling TLS for the DAV blobstore. Refer to [bosh-deployment](https://github.com/cloudfoundry/bosh-deployment) for a full manifest reference.
+```
+...
+instance_groups:
+- name: bosh
+  properties:
+    agent:
+      env:
+        bosh:
+          blobstores:
+          - provider: dav
+            options:
+              endpoint: https://((internal_ip)):25250
+              user: agent
+              password: ((blobstore_agent_password))
+              tls:
+                cert:
+                  cat: ((blobstore_ca.certificate))
+
+    blobstore:
+      address: ((internal_ip))
+      port: 25250
+      provider: dav
+      director:
+        user: director
+        password: ((blobstore_director_password))
+      agent:
+        user: agent
+        password: ((blobstore_agent_password))
+      tls:
+        cert:
+          ca: ((blobstore_ca.certificate))
+          certificate: ((blobstore_server_tls.certificate))
+          private_key: ((blobstore_server_tls.private_key))
+...
+```
+
+
+## Ignored or Stopped Instances
+
+Take note of any ignored instances with `bosh instances --details`. These instances will not be considered when redeploying or recreating instances. Ignored instances will only receive new certificates when recreated.
+
+Any instances stopped with `bosh stop` will not receive the new CA or certificate until recreated. Performing `bosh start` on these instances will transition the VM to running successfully. However, if a redeploy does not trigger a recreate, this instance will fail to update. The agent consults the blobstore for templates during VM lifecycle changes.
+
+Instances that have been stopped with `bosh stop --hard` have been destroyed, keeping the persistent disk if applicable. Performing a `bosh start` on those instances will recreate the VM. These instances will then receive the new CA.
+
+## Introducing a New Certificate Authority (CA) **(for smaller environments)**
```

Comment:
We think this simplified strategy still has merit to keep. It will allow operators to work with less downtime, as it requires half of the recreates. This may still be advantageous for users.